configCivcraft.yml

```diff
@@ -4746,29 +4746,29 @@ recipes:
     input:
       poisonouspotatoes:
         material: POISONOUS_POTATO
-        amount: 64
+        amount: 128
       enderchests:
         material: ENDER_CHEST
-        amount: 32
+        amount: 64
       diamondblocks:
         material: DIAMOND_BLOCK
-        amount: 16
+        amount: 32
```

Comment:
Not necessarily saying this is too much, but be aware that this equals 2 rows of end2.0 diamond blocks